src/main/resources/assets/skyblocker/lang/en_us.json

```diff
@@ -1046,9 +1046,13 @@
   "skyblocker.config.uiAndVisuals.slotText.enchantmentLevel": "Enchantment Book Level",
   "skyblocker.config.uiAndVisuals.slotText.essenceShop": "Essence Shop",
   "skyblocker.config.uiAndVisuals.slotText.essenceShop.@Tooltip": "Displays the levels and your current amount of essence while in essence shops.",
+  "skyblocker.config.uiAndVisuals.slotText.evolvingItems": "Evolving Items",
+  "skyblocker.config.uiAndVisuals.slotText.evolvingItems.@Tooltip": "Displays the time held for Evolving Items (Bottle of Jyrre, Moby Duck, Discrite, Dark Cacao Truffle, Training Weights)",
```

Comment:
Maybe remove "Evolving Items" and the parenthesis here, so it's just "Displays the time held for Bottle of Jyrre, Moby Duck, Discrite, Dark Cacao Truffle, and Training Weights."